src/ppmdu/pmd2/pmd2.cpp

```diff
@@ -76,8 +76,13 @@ namespace pmd2
         "Spanish",
     }};
 
-    const toolkitversion_t PMD2ToolsetVersionStruct{0,1,0};
-
+    /*
+    Current version string. Represents the version of the PPMDU library used.
+    */
+    const std::string PMD2ToolsetVersion = "0.2.0"; //! #TODO: Replace with something that takes care of incrementing this for us!
```

Comment:
This was moved to pmd2.cpp to keep the two version variables right next to each other, reducing the chance the bug will happen again.